docs/transaction-request.md

```diff
@@ -0,0 +1,24 @@
+# TransactionRequest
+
+An object describing a transaction that can get signed by a library like ethers.js or Web3.js.
+
+## Properties
+
+| Name       | Type     | Description                                  |
+| ---------- | -------- | -------------------------------------------- |
+| `chainId`  | `Number` | Chain ID of the network.                     |
+| `data`     | `String` | Transaction data.                            |
+| `gasLimit` | `Number` | Maximum gas this transaction may spend.      |
+| `gasPrice` | `String` | Price (in wei) per unit of gas.              |
+| `to`       | `String` | Target address or ENS name.                  |
+| `value`    | `String` | Amount (in wei) this transaction is sending. |
+
+## Methods
```

Comment:
Does it make sense to also have `toRpc()` for handling the transaction without a library?

Reply:
Absolutely, good idea!